### PR TITLE
chore: upgrade requirements and fix pylint violations

### PR DESCRIPTION
**Pull request by stvstnfrd**
This replaces #186 and fixes the failing tests.

### CODE REVIEW

File: lti_consumer/lti_1p3/extensions/rest_framework/serializers.py [hunk 1]
```diff
@@ -19,12 +19,11 @@ class UsageKeyField(serializers.Field):
     from the platform like so:
     `from openedx.core.lib.api.serializers import UsageKeyField`
     """
-    # pylint: disable=arguments-differ
-    def to_representation(self, data):
+    def to_representation(self, value):
```
Comment (sarina) [inline lti_consumer/lti_1p3/extensions/rest_framework/serializers.py:22]: what was the reasoning behind naming the arg `data` ? I assume that changing it to `value` doesn't affect anything

**Review by sarina — APPROVED** [review]

File: lti_consumer/lti_1p3/extensions/rest_framework/serializers.py (hunk 1 above)
Comment (stvstnfrd) [reply lti_consumer/lti_1p3/extensions/rest_framework/serializers.py:22]: This changed "upstream" in the parent class; we get a warning without the change (though it technically shouldn't break _unless_ it was mistakenly called with a `kwarg`)

Comment (stvstnfrd) [reply lti_consumer/lti_1p3/extensions/rest_framework/serializers.py:22]: Slightly more context in the commit message: https://github.com/edx/xblock-lti-consumer/pull/188/commits/1e17937cb0532de1d26dfda2da93487b31c137ec

Comment (sarina) [reply lti_consumer/lti_1p3/extensions/rest_framework/serializers.py:22]: 👍  ty

**Pull request merged**